.travis.yml

```diff
@@ -1,41 +1,54 @@
 ---
 language: node_js
 node_js:
-  - "4"
+  # we recommend testing addons with the same minimum supported node version as Ember CLI
+  # so that your addon works for all apps
+  - "6"
 
 sudo: false
+dist: trusty
+
+addons:
+  chrome: stable
 
 cache:
   directories:
     - $HOME/.npm
-    - $HOME/.cache # includes bowers cache
 
 env:
-  # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-1.13
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
-  - EMBER_TRY_SCENARIO=ember-release
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
-
-matrix:
-  fast_finish: true
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+
+jobs:
+  fail_fast: true
   allow_failures:
-    - env: EMBER_TRY_SCENARIO=ember-1.13
     - env: EMBER_TRY_SCENARIO=ember-canary
-    
+
+  include:
+    # runs linting and tests with current locked deps
+
+    - stage: "Tests"
+      name: "Tests"
+      script:
+        - npm run lint:hbs
+        - npm run lint:js
+        - npm test
+
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - stage: "Additional Tests"
+      env: EMBER_TRY_SCENARIO=ember-lts-2.16
```

Comment:
I went and accepted the new "default" testing versions -- if you want to maintain the tests being run against Ember 1.13 and 2.4, I'm happy to go and add those to `ember-try` again